android/src/main/java/com/usabilla/reactnative/ubform/UsabillaBridge.java

```diff
@@ -70,9 +71,9 @@ public String getName() {
     @ReactMethod
     public void initialize(@NonNull String appId) {
         final Activity activity = getCurrentActivity();
-        if (activity != null) {
-            usabilla = Usabilla.getInstance(activity.getBaseContext());
+        if (activity != null) {  
             usabilla.initialize(activity.getBaseContext(), appId);
+            usabilla.updateFragmentManager(((FragmentActivity) activity).getSupportFragmentManager());
```

Comment:
Shouldn't there be a error message if the  usabilla.initialize(activity.getBaseContext(), appId). fails, alerting the user/developer that appID might be wrong

Reply:
That would definitely be great to have!

Reply:
There is :)
It's not done in the bridge but in the SDK itself, when the `appId` passed is not valid the error is logged.
The only thing is that at the moment the log is internal and not external for the user, but that should be fixed in the SDK not in a RN PR